FastSurferCNN/README.md

```diff
@@ -20,14 +20,15 @@ The *FastSurferCNN* directory contains all the source code and modules needed to
 * `--strip`: strip suffix from path definition of input file to yield correct subject name. (Optional, if full path is defined for `--t1`)
 * `--lut`: FreeSurfer-style Color Lookup Table with labels to use in final prediction. Default: ./config/FastSurfer_ColorLUT.tsv
 * `--seg`: Name of intermediate DL-based segmentation file (similar to aparc+aseg).
-* `--cfg_cor`: Path to the coronal config file
-* `--cfg_sag`: Path to the axial config file
-* `--cfg_ax`: Path to the sagittal config file
 
-#### Checkpoints
+#### Checkpoints and configs
 * `--ckpt_sag`: path to sagittal network checkpoint
 * `--ckpt_cor`: path to coronal network checkpoint
 * `--ckpt_ax`: path to axial network checkpoint
+* `--cfg_cor`: Path to the coronal config file
+* `--cfg_sag`: Path to the axial config file
+* `--cfg_ax`: Path to the sagittal config file
+* `--plane_yaml` Path to the yaml file, containing url, ckpt and cfg information
```

Comment:
I am not sure about making this a parameter .... these are default values.

Reply:
The idea was, that it could be easier to pass a file with other ckps and cfgs, as an alternative to the somewhat long flags. But this might be a niche case and not neccesarily adds a lot of benefit. I could revert it back if you want.

Reply:
But the information in the yaml file (the file we pass here) is all static information. There is no reason to change this for users.
This change primarily for us to dis-aggregate code and content/configuration/data.